clusterloader2/pkg/proxypod/manifest/proxy_pod.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+spec:
+  containers:
+  - name: ubuntu
+    image: ubuntu:latest
+    command: [ "/bin/bash", "-c", "--" ]
+    args: [ "apt-get update; apt-get install --assume-yes curl; while true; do sleep 30; done;" ]
```

Comment:
Can't we really take something existing?

Reply:
I changed image to "agnhost:2.2" which is used to execute curl in kubernetes e2e tests.